packages/dashmate/src/docker/DockerCompose.js

```diff
@@ -39,7 +41,7 @@ class DockerCompose {
   async runService(envs, serviceName, command = [], options = []) {
     await this.throwErrorIfNotInstalled();
 
-    if (await this.isServiceRunning(envs, serviceName)) {
+    if (await this.isNodeRunning(envs, serviceName)) {
```

Comment:
It's little bit wired to pass service name to `isNodeRunning` function. I would recommend to have two methods:
- `isNodeRunning` which will rely on compose files
- `isServiceRunning` which will accept service name (not optional)